fed-budget-chart/css/lines.css

```diff
@@ -0,0 +1,210 @@
+:root {
+  --color: #000;
+}
+
+body {
+  font-family: "Source Sans Pro", Arial, sans-serif;
+  line-height: 1.3;
+}
+
+p {
+  margin: 0;
+}
+
+p + p {
+  margin-top: 1rem;
+}
+
+#outerContainer {
+  display: grid;
+  grid-gap: .5rem;
+  grid-template-columns: 1fr;
+  align-items: start;
+  max-width: 1600px;
+  margin: 0 auto;
+}
+
+.highcharts-background {

```

Comment:
What's this for?

Reply:
that's a good question! it's not in my current code so i must have taken it out after submitting the PR.